README.md

```diff
@@ -442,7 +442,62 @@ module.exports = function(app) {
   });
 };
 ```
+#### Breaking Changes with GeoPoint since 5.x
+Previous to `loopback-connector-mysql@5.x`, MySQL connector was saving and loading GeoPoint properties from the MySQL database in reverse.
```

Comment:
sorry.. not sure if that's more accurately in english.. should it be "Prior to"?